### PR TITLE
Improved installation, CI and integration with updated online_monitor package

### PR DESCRIPTION
**Pull request by leloup314**
This PR largely simplifies the installation process by adjusting the requirements. Furthermore it updates the installation process to use the updated online-monitor package. The installation section in the readme is updated accordingly. Also the testing on github actions is massively simplified, using more actions, removing the need for [custom CI imports](https://github.com/SiLab-Bonn/pymosa/blob/master/pymosa/tests/test_monitor.py#L85-L89), and is easier to maintain.

### CODE REVIEW

**Comment by leloup314** [comment]
Ready from my side!

File: setup.py [hunk 1]
```diff
@@ -35,18 +35,3 @@
...
-    import pymosa
-    from online_monitor.utils import settings
-    # Get the absoulte path of this package
-    package_path = os.path.dirname(pymosa.__file__)
-    # Add online_monitor plugin folder to entity search paths
-    settings.add_producer_sim_path(os.path.join(package_path, 'online_monitor'))
-    settings.add_converter_path(os.path.join(package_path, 'online_monitor'))
-    settings.add_receiver_path(os.path.join(package_path, 'online_monitor'))
-except (ImportError, pkg_resources.DistributionNotFound):
-    pass
```
Comment (YannickDieter) [inline setup.py:52]: Why did you remove this?

Comment (YannickDieter) [reply setup.py:52]: Okay, understood.
